pkg/internal/local/commands/common.go

```diff
@@ -77,3 +84,36 @@ func validateFlagInputs(flags *pflag.FlagSet) error {
 	}
 	return nil
 }
+
+func createHostPathConfig() (string, error) {
+	pluginDir, ok := os.LookupEnv(utils.PluginDirectoryPath)
+	if !ok {
+		return "", nil
+	}
+	if strings.TrimSpace(pluginDir) == "" {
+		return "", nil
+	} else {
```

Comment:
We can reduce indentation by loosing the `else`-block.

Reply:
done